discovery_engine_core/web-api/src/migrations/20220926145400_document.sql

```diff
@@ -0,0 +1,24 @@
+--  Copyright 2022 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+CREATE TABLE IF NOT EXISTS document (
```

Comment:
it's not really a document, maybe a `document_interaction(_log)` might be a better name? wdyt?

Reply:
i'm fine with any suggested name, i'll change it to `interaction` for now